packages/live-status-gateway/src/topics/adLibsTopic.ts

```diff
@@ -34,6 +34,7 @@ interface AdLibActionType {
 
 interface AdLibStatus {
 	id: string
+	partId?: string
```

Comment:
Same as above, this isn't exactly an optional property - there's just two types of objects, one with an one without `partId`, so maybe this should be split into two interfaces: `AdLibStatus` and `GlobalAdLibStatus`?